app/models/concerns/hostext/power_interface.rb

```diff
@@ -0,0 +1,28 @@
+module Hostext
```

Comment:
Just `host` suffices, not sure why the other one (search) has always been `Hostext`, I guess because it deals with searching text.. I can fix this on merge though.